helm_deploy/cla-backend-migration/templates/deployment.yaml

```diff
@@ -23,40 +23,40 @@ spec:
             - name: TARGET_DB_HOST
               valueFrom:
                 secretKeyRef:
-                  name: database-14
+                  name: eligibility-integration-database-14
```

Comment:
Can this please be reverted before merging

Reply:
Agreed, it's on our list to revert (see: PR description "TODO before merge")